apps/full-site-editing/full-site-editing-plugin/donations/index.js

```diff
@@ -81,7 +82,7 @@ const settings = {
 			default: __( 'Donate yearly', 'full-site-editing' ),
 		},
 	},
-	category: 'common',
+	category: getCategoryWithFallbacks( 'widgets', 'common' ),
```

Comment:
Should become just `widgets`, (matching legacy/updated category).

Reply:
>     1. Newspack Blocks (`Blog Posts` & `Post Carousel`) seem to always get the `widgets` category, regardless of the Gutenberg version (pre/post category names change).
> 

[Done.](https://github.com/Automattic/wp-calypso/pull/43670/commits/f9fec7ae017b908e128d498af11b98ea9b1c95ba)